app/models/authentication_request_header.rb

```diff
@@ -0,0 +1,14 @@
+class AuthenticationRequestHeader < Authentication
+
+  def ansible_config_format
+    options = {}
+    options["challengeURL"] = request_header_challenge_url
+    options["loginURL"] = request_header_login_url
+    options["clientCA"] = certificate_authority
+    options["headers"] = request_header_headers
+    options["request_header_preferred_username_headers"] = request_header_preferred_username_headers
+    options["request_header_name_headers"] = request_header_name_headers
+    options["request_header_email_headers"] = request_header_email_headers
+    super options
```

Comment:
This building of the hash is not necessary, because it can just be a Hash literal. But if you're doing it this way to keep consistent with other `ansible_config_format` methods where you can't use a Hash literal, then I guess this is ok.